jobs/kubelet/templates/bin/drain.erb

```diff
@@ -11,6 +11,11 @@ PIDFILE=/var/vcap/sys/run/kubernetes/kubelet.pid
 exit_code=1
 trap ensure_safe_exit EXIT
 
+node_name="<%= spec.ip %>"
+<% if p("cloud-provider.type") == "gce" %>
+  node_name="$(curl http://metadata.google.internal/computeMetadata/v1/instance/name -H "Bearer: $(curl "http://metadata.google.internal/computeMetadata/v1/instance/service-accounts/default/token" -H "Metadata-Flavor: Google" | jq '.access_token')" -H "Metadata-Flavor: Google").c.<%= p("cloud-provider.gce.project-id") %>.internal"
```

Comment:
Thanks! I updated it to remove the call to get the token. It looks a little nicer now.